.github/workflows/module-github-project.yaml

```diff
@@ -0,0 +1,16 @@
+name: module-github-project
+
+on:
+  workflow_call:
+
+jobs:
+  project:
+    runs-on: ubuntu-latest
+    steps:
+      # TODO: switch to main
+      #- uses: cloudeteer/actions/manage-github-project@main
```

Comment:
this can be resolved as soon as the feature branch `cloudeteer/actions/manage-github-project@add-manage-github-project` is merged to main.